buildSrc/src/main/kotlin/blogpost/DownloadGradleProfileTask.kt

```diff
@@ -21,7 +21,7 @@ abstract class DownloadGradleProfileTask @Inject constructor(
 ) : DefaultTask() {
 
     @get:OutputDirectory
-    val gradleProfilerDir: Provider<Directory> = projectLayout.buildDirectory.dir("gradle-profiler")
+    val gradleProfilerDir: Provider<Directory> = projectLayout.buildDirectory.dir("gradle-profiler-$GRADLE_PROFILER_VERSION")
```

Comment:
version needed to be part of the output, otherwise the task wont be rerun if the version of gradle-profiler is changed